theme/less/_homepage.less

```diff
@@ -0,0 +1,95 @@
+.homepage div
```

Comment:
We have a problem here. When I removed the `.homepage` the page showed grid configurations of `_main.less`  (particularly `body > div`) because we import `_main.less`  in `homepage.less`. We need to separate those  configurations in other file too or to move it to `main.less`. Also we can have one `homepage.less` unified with `_homepage.less`. And of course have a `_generic.less` for general properties.

Reply:
> When I removed the .homepage the page shows grid configurations of _main.less (particularly body > div) because we import _main.less in homepage.less

What I'm trying to say is that you don't have to introduce another class to change those predefined values, so `body > div ` can also work, or loading default layout styles only for default layout.

Reply:
What about `_content.less`?

Reply:
> _content.less

What about it? Not sure if I'm following.